streamlit_app_simple.py

```
@@ -0,0 +1,28 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)